docs/command_line_reference.rst

```diff
@@ -272,6 +272,26 @@ A unique identifier for an installation. This id is automatically generated by R
 
 Used to specify the current node's name in the cluster when it is setup via the ``install`` subcommand.
 
+``cluster-name``
+~~~~~~~~~~~~~~~~
+
+.. warning::
+
+    This command line parameter is experimental. Expect the functionality and the command line interface to change significantly even in patch releases.
+
+This parameter is useful in benchmarks involved multiple Elasticsearch clusters. It's used to configure the cluster name of the current Elasticsearch node when it is setup via the ``install`` or ``race`` subcommand. The following example setups two Elasticsearch clusters: ``cluster-1`` and ``cluster-2``, and each has two nodes::
```

Comment:
Nit: change "setups" to "sets up".

Reply:
thanks, fixed in https://github.com/elastic/rally/pull/1478/commits/538418d0e19670e7efcf6d6e8528c8cc3eb4ffe3